examples/jquery/main.js

```diff
@@ -9,7 +9,9 @@ function toggleTrait () {
 }
 
 function login () {
-    flagsmith.identify("flagsmith_sample_user");
+    flagsmith.identify("flagsmith_sample_user", {
+        createdAt: undefined
```

Comment:
was this here for testing purposes or do we want to keep it?

Reply:
Will revert